server/src/main/java/org/elasticsearch/action/support/replication/ReplicationResponse.java

```diff
@@ -50,9 +50,10 @@
 
     private ShardInfo shardInfo;
 
-    @Override
-    public void readFrom(StreamInput in) throws IOException {
```

Comment:
it has been common practice to leave this method implemented and have it throw an exception when called. Do you think this is unnecessary?